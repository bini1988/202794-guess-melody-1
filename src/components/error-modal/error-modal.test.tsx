import React from "react";
import renderer from "react-test-renderer";
import ErrorModal from "./error-modal";


it(`ErrorModal correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <ErrorModal/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
