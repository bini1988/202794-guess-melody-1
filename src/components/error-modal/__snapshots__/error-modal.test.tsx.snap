// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorModal correctly renders default markup 1`] = `
<section
  className="modal"
>
  <h2
    className="modal__title"
  >
    Произошла ошибка!
  </h2>
  <p
    className="modal__text"
  >
    Статус: 404. Пожалуйста, перезагрузите страницу.
  </p>
</section>
`;
